templates/wordpress/files/composer.json

```diff
@@ -31,20 +31,18 @@
         "sort-packages": true
     },
     "scripts": {
-      "update-packages": [
-          "cp -rf wp/* web/",
-          "rm -rf wp"
+      "movewpcore-todocroot": [
+          "cp -rf wordpress/* web/ && rm -rf wordpress"
```

Comment:
Shouldn't this be just a "mv" command...?

Reply:
@Crell `mv wordpress/* web` fails because our template upstream contains `wp-content/`.

```
> mv wordpress/* web
mv: rename wordpress/wp-content to web/wp-content: Directory not empty
```

Which has me thinking... is there a reason we need anything other than `wp-config.php` in `web` to begin with, if all we're doing after install is copying everything over? It looks like it doesn't affect your ability to download plugins/themes. All that's in there now it READMEs, and those subdirectories are a apart of the upstream anyways.